.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto-Dependabot
+
+on: 
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kjvalencik/github-actions/approve@master
+        if: github.event_name == 'pull_request' && (github.actor == 'dependabot-preview[bot]' || github.actor == "dependabot[bot]')
```

Comment:
```suggestion
        if: github.event_name == 'pull_request' && (github.actor == 'dependabot-preview[bot]' || github.actor == 'dependabot[bot]')
```
Syntax error :sweat_smile: 
![image](https://user-images.githubusercontent.com/9273484/63367059-4d7de200-c37b-11e9-9790-96e2937eebdb.png)

Reply:
Just pushed the fix!